src/group_gui.cpp

```diff
@@ -862,6 +915,16 @@ class VehicleGroupWindow : public BaseVehicleListWindow {
 				}
 				break;
 			}
+			case WID_GL_CREATE_GROUP: { // make new group with vehicle specific name and add vehicle
```

Comment:
i feel like this comment should be part of the documentation of the enum value, rather than here

Reply:
I put it to there, but I am not sure that's the right place.